STYLEGUIDE.md

```diff
@@ -217,6 +217,120 @@ Default: http://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml
     templates include a script tag which assigns app_options to
     a Javascript variable so that it as accessible from JS.
 
+### React
+* <a name="js-react-inline-styles"></a>
+Prefer single object for all styles vs. inlined style objects
+```jsx
+// Bad
+var component = (
+  <div style={{color: 'red', display: 'block'}}>
+    <div style={{color: 'blue', fontSize: 10}}>I'm a child</div>
+  </div>
+);
+
+// Good
+var styles = {
+  root: {
+    color: 'red',
+    display: 'block
+  },
+  child: {
+    color: 'blue',
+    fontSize: 10
+  }
+};
+...
+var component = (
+  <div style={styles.root}>
+    <div style={styles.child}>I'm a child</div>
+  </div>
+);
+```
+* <a name="js-react-pixel-numbers"></a>
+Prefer numbers vs strings for pixel values
+```jsx
+// Bad
+var styles = {
+  root: {
+    width: '100px',
+    height: '100px'
+  }
+}'
+
+// Good
+var styles = {
+  root: {
+    width: 100,
+    height: 100
+  }
+};
```

Comment:
We could say that for simple styles (only one field, or maybe two?) it's okay to inline? What do folks thing?